features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/Workflow/APIStateChange.jsx

```diff
@@ -0,0 +1,471 @@
+            .catch((e) => {
+                Alert.error(e);
+            });
+    };
+
+    useEffect(() => {
+        fetchData();
+    }, []);
+
+
+    const updateStatus = (referenceId, value) => {
+        const body = {};
+        body.status = value;
+        body.attributes = {};
+        body.description = 'Approve workflow request.';
+
+        let promisedupdateWorkflow = '';
+        promisedupdateWorkflow = restApi.updateWorkflow(referenceId, body);
+        return promisedupdateWorkflow
+            .then(() => {
+                return (
+                    <FormattedMessage
+                        id='Workflow.ApplicationCreation.update.success'
+                        defaultMessage='workflow status is updated successfully'
+                    />
+                );
+            })
+            .catch((error) => {
+                const { response } = error;
+                if (response.body) {
+                    response.body.description = 'Unable to complete API state change approve/reject process';
+                    throw (response.body.description);
+                }
+                return null;
+            })
+            .finally(() => {
+                fetchData();
+            });
+    };
+
+    const pageProps = {
+        help: (
+            <HelpBase>
+                <List component='nav' aria-label='main mailbox folders'>
+                    <ListItem button>
+                        <ListItemIcon>
+                            <DescriptionIcon />
+                        </ListItemIcon>
+                        <Link
+                            target='_blank'
+                            href={Configurations.app.docUrl
+                        + 'learn/api-microgateway/grouping-apis-with-labels/#grouping-apis-with-microgateway-labels'}
+                        >
+                            <ListItemText primary={(
+                                <FormattedMessage
+                                    id='Workflow.APIStatechange.help.link.one'
+                                    defaultMessage='Create a API State change workflow request'
+                                />
+                            )}
+                            />
+
+                        </Link>
+                    </ListItem>
+                </List>
+            </HelpBase>),
+        /*
+        pageStyle='half' center part of the screen.
+        pageStyle='full' = Take the full content area.
+        pageStyle='paperLess' = Avoid from displaying background paper. ( For dashbord we need this )
+        */
+        pageStyle: 'half',
+        title: intl.formatMessage({
+            id: 'Workflow.APIStateChange.title.apistatechange',
+            defaultMessage: 'API State Change - Approval Tasks',
+        }),
+    };
+
+    const classes = useStyles();
+    const [searchText, setSearchText] = useState('');
+
+    const columProps = [
+        {
+            name: 'description',
+            label: 'Description',
```

Comment:
Add Internationalisation to the labels of coloumn props

Reply:
fixed with 54dd617